docs/release-notes/eclair-v0.9.0.md

```diff
@@ -0,0 +1,312 @@
+# Eclair v0.9.0
+
+This release contains a lot of preparatory work for important (and complex) lightning features: dual-funding, splicing and Bolt 12 offers.
+Those features are now fully implemented in `eclair`, but we're waiting for the specification work to be finalized and other implementations to be ready for cross-compatibility tests.
```

Comment:
```suggestion
These features are now fully implemented in `eclair`, but we're waiting for the specification work to be finalized and other implementations to be ready for cross-compatibility tests.
```

Reply:
Thanks, done in https://github.com/ACINQ/eclair/pull/2693/commits/8085804cdaab47ef34ad146c5332f7ab7f131ccf